cheatsheets/LDAP_Injection_Prevention_Cheat_Sheet.md

```diff
@@ -105,6 +105,25 @@ occur before the validation step (source: [Prevent LDAP injection](https://wiki.
 
 For further information visit [OWASP ESAPI Java Encoder Project which includes encodeForLDAP(String) and encodeForDN(String)](https://owasp.org/www-project-java-encoder/).
```

Comment:
Just noticed this from the context.  OWASP ESAPI is a different library than the OWASP Java Encoder Project. The 2 methods that are mentioned are only in ESAPI. Instead, this should say:

> For further information visit [OWASP ESAPI](https://owasp.org/www-project-enterprise-security-api/) which includes [encodeForLDAP(String)](https://javadoc.io/static/org.owasp.esapi/esapi/2.7.0.0/org/owasp/esapi/Encoder.html#encodeForLDAP-java.lang.String-) and [encodeForDN(String)](https://javadoc.io/static/org.owasp.esapi/esapi/2.7.0.0/org/owasp/esapi/Encoder.html#encodeForDN-java.lang.String-)

Reply:
Ah; never mind. Didn't notice this was already merged. Getting ready to hit the pillow. @mackowski - If I remember I'll create a issue and maybe a PR, depending on if I can find the time.